src/DIRAC/Core/scripts/dirac-info.py

```diff
@@ -1,72 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I've only modified three scripts for now as an example so people can give feedback without needing to duplicate the work. While the diff looks scary the actual change is:

* Rename `dirac-info.py` to `dirac_info.py` so it's a valid Python module. This is safe as dirac-install already replaces `_` with `-` when generating the wrapper scripts. (I was going to suggest implementing this but it was already there.)
* Indent almost all lines so they can exist in a `main()` function
* Add `if __name__ == "__main__": main()` to the end so the script can still be ran directly